static/app/components/searchQueryBuilder/tokens/filter/specificDatePicker.tsx

```diff
@@ -102,109 +104,111 @@ function SpecificDatePicker({
   const hasTime = Boolean(parsedToken?.time);
 
   return (
-    <SearchBarDatePickerOverlay
-      data-test-id="specific-date-picker"
-      ref={popoverRef}
-      // Otherwise clicks will propagate to the grid and close the dropdown
-      onClick={e => e.stopPropagation()}
-    >
-      {isOpen ? (
-        <Fragment>
-          <DatePicker
-            date={internalDate}
-            onChange={newDate => {
-              if (newDate instanceof Date) {
-                handleSelectDateTime(
-                  createDateStringFromSelection({
-                    date: getInternalDate(newDate, utc),
-                    time,
-                    utc,
-                  })
-                );
-              }
-            }}
-          />
-          <ControlsWrapper>
-            <CheckboxLabel>
-              <Checkbox
-                checked={hasTime}
-                onChange={e => {
-                  if (e.target.checked) {
-                    handleSelectDateTime(
-                      createDateStringFromSelection({
-                        date: internalDate,
-                        time: DEFAULT_DAY_START_TIME,
-                        utc,
-                      })
-                    );
-                  } else {
-                    handleSelectDateTime(
-                      createDateStringFromSelection({
-                        date: internalDate,
-                        utc: true,
-                      })
-                    );
-                  }
-                }}
-              />
-              {t('Include time')}
-            </CheckboxLabel>
-            <TimeUtcWrapper>
-              <TimeInput
-                disabled={!hasTime}
-                time={time ?? DEFAULT_DAY_START_TIME}
-                setTime={newTime => {
+    <StyledPositionWrapper {...overlayProps} visible={isOpen}>
```

Comment:
Hide whitespace and you'll need that I am just wrapping this component in the position wrapper div